library/src/scala/tasty/reflect/Printers.scala

```diff
@@ -1367,6 +1367,9 @@ trait Printers
             case tpe @ Type.SymRef(IsClassSymbol(sym), _) if sym.name.endsWith("$") =>
               printType(tpe)
               this += ".type"
+            case tpe @ Type.SymRef(sym, _) if !IsTypeSymbol.unapply(sym).nonEmpty =>
```

Comment:
`case tpe @ Type.SymRef(IsValSymbol(sym), _) =>` ?

Reply:
`!IsTypeSymbol.unapply(sym).nonEmpty` --> `IsTypeSymbol.unapply(sym).isEmpty`

Reply:
we should probably add `isType` to `SymbolAPI` in `SymbolOps` to make this simpler to write.